### ISSUE TITLE
Aether Great Trees and Aether Forests

### ISSUE THREAD

**Issue by DeathShadez** [dfsek/BetterEnd-Public#7]
**Is your feature request related to a problem? Please describe.**
Nope.

**Describe the solution you'd like**
There would be a slight chances of single _Aether Great Tree_ growing in both _Aether_ and _Aether Highland_ biomes. In regular _Aether_ biomes, it would be an _Great Oak Tree_ whereas in _Aether Highlands_, it would  _Great Spruce Tree_. These _Great Aether Trees_ are of a moderate size and look pretty swell as they are great and wise-looking I guess. There could also be a biome called the _Aether_Forest_ that would have a load of these _Great Aether Trees_. The trees in the _Aether_Forest_ should probably slightly smaller than Great Aether Trees. The configuration to change the chances of the trees to spawn in regular aether biomes and of the _Aether_Forest_ biome chance rate can be a feature implemented afterwards whenever possible.

**Describe alternatives you've considered**
Change the names as you see fit along with how the trees that spawn will look in shape and size. I expect the forest trees to probably not be as big as the lonely Great Trees or maybe not and have these giant creepy woods lol.

**Additional context**
You can probably implement procedural generating in some fashion for this as you've done with the _Shattered_Forest_ for the _Aether_Forest_. 

I used the Fractual Forest Plugin to help make some reference pictures along with making some by hand. I put both compact and noncompact for a feel of either more spaced out trees or very clustered forests.

TLDR: Random low chance of spawning _Great Aether Trees_ (Structure?) in aether biomes and _Aether_Forest_ (biome).

Pictures below for a feel for how it would sorta look

### Aether Great Trees
**Great Oak Tree**
![Lonely Oak Tree ;(](https://user-images.githubusercontent.com/51402617/84248229-25609a80-aad7-11ea-8737-aa8fdbc8a6dc.png)
Note: Low chance of spawning in regular Aether biomes.

**Great Spruce Tree**
![Lonely Spruce tree](https://user-images.githubusercontent.com/51402617/84248587-a61f9680-aad7-11ea-8ee3-70dc2c31c9e1.png)
Note: Low chance of spawning in regular Aether Highlands biomes. Handmade the tree...yikes.

### Aether Great Forest Samples
Note: Took multiple references depending on how this biome may be planned on being generated to look about

**Aether Great Oak Forest Compact**
![compact forest](https://user-images.githubusercontent.com/51402617/84248873-0282b600-aad8-11ea-86cd-320b210c93a6.png)
Note: Compact version of a Aether Forest biome in regular looking aether biomes

**Aether Great Oak Forest Non-Compact
![noncompact forest](https://user-images.githubusercontent.com/51402617/84248945-17f7e000-aad8-11ea-8b61-1603745e638a.png)
Note: Non-compact of a Aether Forest biome in regular looking aether biomes as they are more spaced out and less clustered

**Aether Great Spruce Forest**
![Spruce forest](https://user-images.githubusercontent.com/51402617/84248712-cfd8bd80-aad7-11ea-93a7-d0e54134be2d.png)
Note: Forest biome in Aether highlands sorta deal with spruce trees instead of oak, so it may be an another biome all together like the Aether_Highlands_Forest or something shorter. All duplicates of one another, but just trying to give a reference.

Adapt or change this idea to how you seek fit or what not. Along with saying what a job well done for all you've put into this plugin!

**Comment by dfsek** [comment]
Working on this now; will most likely make each variant a separate biome.

**Comment by dfsek** [comment]
Released in 3.1.0 :)
